stack/__main__.py

```diff
@@ -1081,5 +1125,16 @@ def find_service_account(kind: str, access_level: str) -> Optional[str]:
         )
 
 
+def _get_name_from_external_sa(email: str, suffix='.iam.gserviceaccount.com'):
+    """
+    Convert service account email to name + some filterin
+
+    """
```

Comment:
```suggestion
    """Convert service account email to name + some filtering."""
```

Reply:
Thank you! I added whitespace for doctest, but forgot to add it (added now) :)